packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -107,11 +107,17 @@ export class RpcClient implements RpcClientInterface {
     protected url: string,
     protected chain: string = defaultChain,
     protected httpBackend: HttpBackend = new HttpBackend()
-  ) {}
+  ) { }
 
-  protected createURL(path: string) {
```

Comment:
is this something typescript complains? i think it's protected not private for a reason for example if user extend from rpcClient that they can this.createURL but not accessible to public. https://github.com/ecadlabs/taquito/blob/master/docs/cancel_http_requests.md?plain=1#L66

Reply:
Good catch. A result of copy-pasting. Will fix